src/main/java/blackjack/domain/card/Cards.java

```diff
@@ -26,51 +26,39 @@ private void validateDuplicate(List<Card> cards) {
     }
 
     public void concat(Cards cards) {
+        List<Card> tempCards = new ArrayList<>(this.cards);
+        tempCards.addAll(cards.getCards());
+        validateDuplicate(tempCards);
         this.cards.addAll(cards.getCards());
-        new Cards(this.cards);
     }
 
-    public int getBestPossible() {
-        int sum = getLowestSum();
-
-        for (Card card : cards) {
-            sum = updateSum(sum, card);
+    public Score getBestPossible() {
+        if (hasAce()) {
+            return Score.soft(extractedScores());
```

Comment:
```suggestion
    public Score getBestPossible() {
        Score score = extractedScores();
        if (hasAce()) {
            return score.soft();
```

이런 형태도 가능하다고 봐요. soft/hard와 같은 도메인 로직들을 static 메서드에 두는게 적당할까? 에 대해서 한 번 생각해보셔도 좋을 것 같네요 :)